### PR TITLE
Update configure-block-at-first-sight-microsoft-defender-antivirus.md

### PR DESCRIPTION
**Pull request by cventour**
The "Validate block at first sight is working" section was pointing to a document on how to validate Cloud protection is working but it was not specific to BaFS. I added a small paragraph that informs the reader to access the Defender Demos page to access the BaFS page.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/security/defender-endpoint/configure-block-at-first-sight-microsoft-defender-antivirus.md [hunk 1]
```diff
@@ -144,7 +144,9 @@ You can confirm that block at first sight is enabled on individual client device
 
 ## Validate block at first sight is working
 
-To validate that the feature is working, follow the guidance in [Validate connections between your network and the cloud](configure-network-connections-microsoft-defender-antivirus.md#validate-connections-between-your-network-and-the-cloud).
+To validate that the feature is working, attempt to download the Block at First Sight sample file from https://demo.wd.microsoft.com/Page/BAFS. To download the file, you will need an Azure AD credential from your organization and a Security Administrator or Global Administrator Azure AD role assigned to that account.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/configure-block-at-first-sight-microsoft-defender-antivirus.md]:
```suggestion
To validate that the feature is working, attempt to download the [Block at First Sight sample file](https://demo.wd.microsoft.com/Page/BAFS). To download the file, you will need an Azure AD credential from your organization and a Security Administrator or Global Administrator Azure AD role assigned to that account.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @cventour, kindly commit the suggested change. Many thanks.

**Review by denisebmsft — APPROVED** [review]
Thank you @cventour

**Pull request merged**